pom.xml

```diff
@@ -217,6 +217,22 @@
         </dependency>
     </dependencies>
     <build>
+		<testResources>
+        	<testResource>
```

Comment:
@pinaf This is how maven resources plugin works; I also find it a bit counter-intuitive. The first one copies all the files except the `keystore` to the target directory replacing all the property placeholders (`filtering=true`). The second one copies only the `keystore` file without processing its contents (`filtering=false`).

An example in the maven official docs can be found [here](http://maven.apache.org/plugins/maven-resources-plugin/examples/filter.html).